### PR TITLE
Ghostly Hollowing process injection rule

### PR DESCRIPTION
**Pull request by sara-rn**
Ghostly Hollowing injection technique described here https://github.com/hasherezade/transacted_hollowing/tree/main#ghostly-hollowing

### CODE REVIEW

**Comment by mike-hunhoff** [comment]
Hi @sara-rn - can you add the referenced example file to https://github.com/mandiant/capa-testfiles?

File: host-interaction/process/inject/process-ghostly-hollowing.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: process ghostly hollowing
```
Comment (mr-tz) [inline host-interaction/process/inject/process-ghostly-hollowing.yml:3]:
```suggestion
    name: use process ghostly hollowing
```

File: host-interaction/process/inject/process-ghostly-hollowing.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    authors:
+      - sara.rincon@mandiant.com
+    scopes:
+      static: function
+      dynamic: call
+    references:
+      - https://github.com/hasherezade/transacted_hollowing/tree/main#ghostly-hollowing
+    examples:
+      - 3b2eba4789bd4a799fe18476a4d1ce9f37ecc4c202eb406e06425c7e792904ff:0x140007aa0
+      - 3b2eba4789bd4a799fe18476a4d1ce9f37ecc4c202eb406e06425c7e792904ff:0x140007840
```
Comment (mr-tz) [inline host-interaction/process/inject/process-ghostly-hollowing.yml:14]: is one example enough here or are the two functions different and both meaningful?

**Review by mr-tz — COMMENTED** [review]
looks good besides the suggestion and question! thank you!

File: host-interaction/process/inject/process-ghostly-hollowing.yml (hunk 2 above)
Comment (mr-tz) [inline host-interaction/process/inject/process-ghostly-hollowing.yml:14]:
```suggestion
      - 3b2eba4789bd4a799fe18476a4d1ce9f37ecc4c202eb406e06425c7e792904ff:0x140007aa0  # open_file
      - 3b2eba4789bd4a799fe18476a4d1ce9f37ecc4c202eb406e06425c7e792904ff:0x140007840  # delete_pending_file
```

**Pull request merged**